File: Archiver/app_code/clsTimeCalcs.vb
Public Class clsTimeCalcs

    Public Function ElapsedTimeInMS(ByVal sTime As Date, ByVal eTime As Date) As Double

        Dim elapsed_time As TimeSpan = Nothing
        elapsed_time = eTime.Subtract(sTime)
        Dim txTotalTime As Double = elapsed_time.Milliseconds
        Return txTotalTime

    End Function

End Class

File: Archiver/app_code/clsTimeCalcs.cs
using System;
using MODI;

namespace EcmArchiver
{
    public class clsTimeCalcs
    {
        public double ElapsedTimeInMS(DateTime sTime, DateTime eTime)
        {
            TimeSpan elapsed_time = default;
            elapsed_time = eTime.Subtract(sTime);
            double txTotalTime = elapsed_time.Milliseconds;
            return txTotalTime;
        }
    }
}